app/filtering.js

```diff
@@ -47,6 +48,8 @@ const headersReceivedFilteringFns = []
 let partitionsToInitialize = ['default', appConfig.tor.partition]
 let initializedPartitions = {}
 
+let torDaemon = new tor.TorDaemon()
```

Comment:
it looks like you don't reassign to this variable so better to use `const` instead of `let`

also this doesn't need to be in the module scope, right? so it can be declared in setupTor instead.

Reply:
done in 453e327b099187651abd294ab409517431991378